samples/scala/src/main/scala/com/microsoft/partnercatalyst/cosmosdb/samples/CountAggregates.scala

```diff
@@ -0,0 +1,41 @@
+package com.microsoft.partnercatalyst.cosmosdb.samples
+
+import com.microsoft.azure.cosmosdb.spark.config.{Config, CosmosDBConfig}
+import com.microsoft.azure.cosmosdb.spark.schema._
+import org.apache.spark.sql.SparkSession
+import org.apache.spark.streaming.{Minutes, StreamingContext}
+import org.apache.spark.{SparkConf, SparkContext}
+
+object CountAggregates {
+
+  def main(args: Array[String]): Unit = {
+    val appName = this.getClass.getSimpleName
+    val conf = new SparkConf()
+      .setAppName(appName)
+      .setIfMissing("spark.master", "local[*]")
+    val sc = new SparkContext(conf)
+    val ssc = new StreamingContext(sc, Minutes(1))
+    val spark = SparkSession.builder().appName(ssc.sparkContext.appName).getOrCreate()
+    import spark.implicits._
+
+    sc.setLogLevel("ERROR")
+
+    val configMap = Map[String, String](
+      CosmosDBConfig.Endpoint -> sys.env("COSMOS_DB_ENDPOINT"),
+      CosmosDBConfig.Masterkey -> sys.env("COSMOS_DB_MASTER_KEY"),
+      CosmosDBConfig.Database -> "samples",
+      CosmosDBConfig.Collection -> "airports",
+      CosmosDBConfig.ConnectionMode -> "Gateway"
```

Comment:
Please note that omitting the `ConnectionMode` entry resulted in an exception like the following:

```
Caused by: com.microsoft.azure.documentdb.DocumentClientException: Server could not parse the Url., StatusCode: NotFound
	at com.microsoft.azure.documentdb.internal.ErrorUtils.maybeThrowException(ErrorUtils.java:69)
	at com.microsoft.azure.documentdb.internal.GatewayProxy.performGetRequest(GatewayProxy.java:240)
	at com.microsoft.azure.documentdb.internal.GatewayProxy.doReadFeed(GatewayProxy.java:109)
	at com.microsoft.azure.documentdb.internal.GatewayProxy.processMessage(GatewayProxy.java:338)
	at com.microsoft.azure.documentdb.DocumentClient$9.apply(DocumentClient.java:2996)
	at com.microsoft.azure.documentdb.internal.RetryUtility.executeDocumentClientRequest(RetryUtility.java:58)
	at com.microsoft.azure.documentdb.DocumentClient.doReadFeed(DocumentClient.java:3006)
	at com.microsoft.azure.documentdb.DocumentQueryClientInternal.doReadFeed(DocumentQueryClientInternal.java:36)
	at com.microsoft.azure.documentdb.internal.query.AbstractQueryExecutionContext.executeRequest(AbstractQueryExecutionContext.java:215)
	at com.microsoft.azure.documentdb.internal.query.DefaultQueryExecutionContext.executeOnce(DefaultQueryExecutionContext.java:131)
	at com.microsoft.azure.documentdb.internal.query.DefaultQueryExecutionContext.fillBuffer(DefaultQueryExecutionContext.java:101)
	at com.microsoft.azure.documentdb.internal.query.DefaultQueryExecutionContext.next(DefaultQueryExecutionContext.java:84)
```